### DISCUSSION TITLE
Searching via Find() method starts from the beginning when result is found inside IFrame

### DISCUSSION THREAD

**Discussion by MaTToX3** [cefsharp/CefSharp#4881] [Q&A]
We implemented a basic search using Find() method, provided by CefSharp. We have a funny issue with it and I am not sure this is part of the design or we are missing something. When calling Find() with findNext parameter set to true and the result is found inside an <iframe>, the next call to Find() will return the very first string found on the whole page - basically it starts searching from the beginning. What is causing such behavior? 

EDIT; what I just found out; if I click to the button that is calling FInd() method with parameter findNExt = true really fast, it works and it goes through the results, found on the iframe. If I stop and wait for a second or two and then click button again, I am back to the first result outside of the iframe. Such behavior does not happen when the result is outside of the iframe.

Hope I explained my issue well enough.

**Comment by amaitland** [comment]
> We implemented a basic search using Find() method, provided by CefSharp

`CefSharp` is just a wrapper. Read the first paragraphs of https://github.com/cefsharp/CefSharp/wiki/General-Usage for background.

> When calling Find() with findNext parameter set to true and the result is found inside an <iframe>, the next call to Find() will return the very first string found on the whole page

Sounds like a bug that needs to be fixed in `CEF`. Raise an issue at https://github.com/chromiumembedded/cef/issues

**Reply by amaitland** [reply]
Make sure you've tested with the current supported version before opening an issue.

`M126` at time of writing.
